test/integration_test/test-deploy.sh

```diff
@@ -533,7 +533,9 @@ kubectl logs stork-test  -n kube-system -f
 for i in $(seq 1 100) ; do
     test_status=$(kubectl get pod stork-test -n kube-system -o json | jq ".status.phase" -r)
     if [ "$test_status" = "Running" ]; then
-        echo "Test is still running, status: $test_status"
+	sleep 5
+        echo "Test is still running, resuming logs from past 5 seconds, status: $test_status"
```

Comment:
nit: Test is still running, status: $test_status, resuming logs from past 10 seconds